components/Chips/src/MDCChipView.m

```diff
@@ -99,67 +119,97 @@ + (Class)layerClass {
 }
 
 - (instancetype)init {
-  if (self = [super init]) {
-    [self commonMDCChipViewInit];
-  }
-  return self;
+  return [self initWithFrame:CGRectZero];
 }
 
 - (instancetype)initWithFrame:(CGRect)frame {
   if (self = [super initWithFrame:frame]) {
```

Comment:
Nit: Although this is absolutely Apple's idiosyncratic approach to initializers, most of the code in MDC-iOS follows the internal 2-line approach:

```objectivec
self = [super initWithFrame:frame];
if (self) {
```